connection.js

```diff
@@ -325,50 +322,35 @@ class Connection {
     }
     // Check challenge sanity checks
     //
-    checkChallengeSanityChecks(notification) {
-
-        // TODO : add all sanity checks here of all unhandled notifications
-        // Sanity check, user can't choose rules. Bots only play chinese.
-        if (!notification.rules.includes("chinese")) {
-            conn_log(`Unhandled rules: ${notification.rules}`);
-            const msg = `The ${notification.rules} rules are not allowed for `
-                        + `this bot, please choose allowed rules such as `
-                        + `chinese rules. `;
-            return { reject: true, msg };
-        }
```

Comment:
Why did we remove this check? Ogs today still enforces Chinese rules for bots right?